src/client/src/ui/analytics/views/analyticsView.jsx

```diff
@@ -43,12 +44,24 @@ export default class AnalyticsView extends ViewBase {
     if (!model.isAnalyticsServiceConnected)
       return <span />;
 
+    let browser = new userAgentParser().getBrowser().name;
```

Comment:
I would pbly add a canPopout method in the popoutService so we dont have to repeat that logic in every view, and then put that on the view model instead of the view itself